### ISSUE TITLE
Leave a call, but you can still hear and talk (not see)

### ISSUE THREAD

**Issue by Nyco** [nextcloud/spreed#3549]
## Steps to reproduce
1. Step in a call where everything works fine: sound, video, screensharing
2. Leave that call

### Expected behaviour
All mics and sounds should be off

### Actual behaviour
I can still hear and talk to participants who are still in the call

## Browser

**Microphone available:** yes

**Camera available:** yes

**Operating system:** Windows/Ubuntu/macOS

**Browser name:** Firefox/Chrome

**Browser version:** Latest

### Browser log

<details>
```
Insert your browser log here, this could for example include:
a) The javascript console log
b) The network log
c) ...
```

</details>

## Talk app

**Talk app version:** (see apps admin page: `/index.php/settings/apps`)

**Custom TURN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/additional`)

**Custom STUN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/additional`)

## Server configuration

**Operating system**: Ubuntu/RedHat/...

**Web server:** Apache/Nginx

**Database:** MySQL/Maria/SQLite/PostgreSQL

**PHP version:**

**Nextcloud Version:** (see admin page)

**List of activated apps:**

<details>
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your server installation folder
```
</details>

**Nextcloud configuration:**

<details>
```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
```
</details>

### Server log (data/nextcloud.log)
<details>
```
Insert your server log here
```
</details>

**Comment by nickvergessen** [comment]
I think this is fixed since #3292

**Comment by Nyco** [comment]
> @nickvergessen nickvergessen added the needs info label 

What info do you need?

**Comment by nickvergessen** [comment]
Whether its fixed with #3292  in master

**Comment by nickvergessen** [comment]
Can't reproduce on current master so I guess #3292 and #3609  helped
